src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -453,5 +453,7 @@ public static string InvalidTokenProviderResponseValue(string invalidValueName)
 
         public const string CombinedUserAppCacheNotSupported = "Using a combined flat storage, like a file, to store both app and user tokens is not supported. Use a partitioned token cache (for ex. distributed cache like Redis) or separate files for app and user token caches. See https://aka.ms/msal-net-token-cache-serialization .";
 
+        public const string JsonParseExceptionB2C = "There might be configuration issues with B2C app or graph scopes are used for B2C application. B2C does not support graph scopes. See inner exception for more details.";

```

Comment:
should this be extended to mention what scopes needs to be passed to fix this issue?

Reply:
Updated the message.